docs/product-ibm-websphere-installation-and-configuration-guide/src/main/asciidoc/download-was-proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='download-was']
+
+= Installing WAS
+
+You must download and install IBM Installation Manager before installing WAS.
+
+== Downloading and installing IBM Installation Manager
+
+. Download IBM Installation Manager version 1.8.5 or later from the http://www-01.ibm.com/support/docview.wss?uid=swg27025142[IBM Installation Manager and Packaging Utility download links] page.
+. Extract the downloaded archive and run the following command as the root user in the new directory:
++
+[source]
+----
+sudo ./install
+----
++
+*IBM Installation Manager* opens.
+. Go to *File* -> *Preferences* and click *Add Repository*.
++
+The *Add Repository* dialog window opens.
+. Enter the repository URL for WAS 9.0. You can find all the repository URLs in the https://www.ibm.com/support/knowledgecenter/SSEQTJ_9.0.0/com.ibm.websphere.installation.nd.doc/ae/cins_repositories.html[Online product repositories for WebSphere Application Server offerings] page of the _IBM Knowledge Center_. For example:
++
+[source]
+----
+http://www.ibm.com/software/repositorymanager/V9WASILAN
+----
+
+== Creating a WebSphere profile and user name
+A profile defines the run time environment. The profile includes all the files that the server processes in the run time environment and that you can change. The user is required for login. In this example, we will use `websphere` for both the user name and password.
+
+. From your terminal, navigate to the WebSphere Application Server folder location that you specified during the installation.
+. Change to the `/bin` directory and run the following command:
++
+[source]
+----
+sudo ./manageprofiles.sh -create -profileName testprofile -adminUserName websphere -adminPassword websphere
+----
+
+== Starting the server
+. From your terminal, navigate to the WebSphere Application Server folder location that you specified during installation.
+. Change to the `/bin` directory and run the following command:
```

Comment:
Same as above, it better to run command from created profile folder.

Reply:
See above comment.

Reply:
Here it can be run from WebSphere profile bin dir.